solutions.js

```diff
@@ -1,5 +1,24 @@
-console.log('Defining functions!');
+// console.log('Defining functions!');
+//
+// function changeNavColor() {
+//   // change the navbar to a different color
+// };
 
-function changeNavColor() {
-  // change the navbar to a different color
-};
+1.
+function changeColor (){
+    topbar.style.backgroundColor = 'red';
+
+}
+changeColor();
+
+2.
+var newElement = document.querySelector('h1');
```

Comment:
Nice job - make sure you're reading the directions all the way though. This one asked for everything to be inside appropriately named functions.